site/package.json

```diff
@@ -21,6 +21,7 @@
     "gatsby-source-filesystem": "2.0.34",
     "gatsby-transformer-remark": "2.3.12",
     "moment": "2.24.0",
+    "netlify-cms": "2.9.5",
```

Comment:
Why is this netlify-cms version required?  Was there specifically something you needed over and above 2.9.1?

Reply:
no longer adding